pkg/kdump/config-client.js

```diff
@@ -264,6 +266,11 @@ export class ConfigFile {
                                 .split(" ")
                                 .filter(e => e != "-F")
                                 .join(" ");
+            } else {
+                settings._internal.core_collector = { value: defaultCoreCollector };
+                if (target.type === "ssh") {
+                    settings._internal.core_collector.value += " -F";
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19434-20231010-092742-cd5300e5-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)